### ISSUE TITLE
Reply-To contains extra characters

### ISSUE THREAD

**Issue by clhammer** [mailgun/mailgun-ruby#40]
When setting the Reply-To with message builder extra characters are applied to the header:

```
mb_obj.add_recipient("h:Reply-To", "test@domain.com");
or
mb_obj.add_custom_parameter("h:Reply-To", "test@domain.com");
```

You end up with:

```
Reply-To[]: test@domain.com
```

This breaks setting a Reply-To as it is not honored by any providers.

**Comment by shaunak-amura** [comment]
Hello @clhammer ,
I had the same problem, this happens because when you use `add_custom_parameter` internally it uses the `complex_setter` method which adds the Reply-To address in an array.
So for handling Reply-To, I used the following code:
`email = Mailgun::MessageBuilder.new`
`email.message["h:Reply-To"] = 'test@domain.com'`

I hope this solves your problem. If you find a better solution, please share, it would be very helpful !

**Comment by pirogoeth** [comment]
Does this still happen in the latest versions of the library?

**Comment by pirogoeth** [comment]
Fixed this in #68.
